File: SelectionSort.java
public class SelectionSort {
    public static void sort(int [] arr){
        int n = arr.length;
        for (int i=0 ;i<n;i++ ){
            int min =arr[i];
            int min_index = i;
            for(int j=i+1;j<n;j++){
            if(arr[j]<min){
                min = arr[j];
                min_index =j;
            }
            }
        
        int swapNum = arr[min_index];
        arr[min_index] = arr[i];
        arr[i] =swapNum;
        }
    }
    public static void main(String[] args) {
      int [] arr = {23,1,34,17,5};
      sort(arr);
      for(int x:arr){
           System.out.print(x + " ");
      }

    }
}

File: SelectionSort.py
def selectionSort(arr):
    n = len (arr)
    for i in range (n):
        min_num = arr[i]
        min_index = i

        for j in range (i+1,n):
            if (arr[j]< min_num):
                min_num =arr[j]
                min_index = j
        temp = arr [min_index]
        arr[min_index] = arr[i]
        arr [i] = temp

a= [56,7,89,43,223,4]
selectionSort(a)
print(a)

